### PR TITLE
Feat: Add fixture files

### PR DESCRIPTION
**Pull request by NSMcEnroe**
Created the fixture files for each of the databases.  I didn't touch the teams file, but both the games and games_teams files have about 45 rows we can play around with.

### CODE REVIEW

File: fixture/teams.csv
Comment (reedhillmar) [inline fixture/teams.csv:1]: Do we need this? It looks like this is a direct copy from /data

File: fixture/game_teams.csv
Comment (reedhillmar) [inline fixture/game_teams.csv:1]: should we name this games_teams_fixture.csv for clarity?

File: fixture/games.csv
Comment (reedhillmar) [inline fixture/games.csv:1]: Same as above

**Review by reedhillmar — CHANGES_REQUESTED** [review]

File: fixture/teams.csv
Comment (NSMcEnroe) [reply fixture/teams.csv:1]: Removed!

File: fixture/game_teams.csv
Comment (NSMcEnroe) [reply fixture/game_teams.csv:1]: Renamed!

File: fixture/games.csv
Comment (NSMcEnroe) [reply fixture/games.csv:1]: Renamed!

**Review by reedhillmar — APPROVED** [review]
Changes look good, I'm happy to merge

**Review by bserge13 — APPROVED** [review]

**Pull request merged**